src/modules/modals/CreateColumn.vue

```diff
@@ -321,7 +328,10 @@ export default {
 				mandatory: this.column.mandatory,
 				viewId: this.isView ? this.element.id : null,
 				tableId: !this.isView ? this.element.id : null,
+				customSettings: this.column.customSettings,
```

Comment:
It will probably look more readable.

Instead of:
`customSettings: this.column.customSettings,`
`data.customSettings = { width: data.customSettings.width }`

Do this here:
`customSettings: { width: this.column.customSettings.width }`